<launch>
  <arg name="frame_id" default="world" />

  <node name="rrt_planner" pkg="rrt_planner" type="rrt_planner_mesh_node" output="screen">
    <remap from="/rrt_planner/pointcloud_traversable" to="/output" />
    <remap from="/input_mesh" to="/surfelmeshing/mesh_pcl" />
    <param name="visualize" value="true" />
    <param name="update_mesh_every_n_sec" value="0.0" />
    <param name="num_seconds_to_plan" value="5.0" />
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="world_frame" value="$(arg frame_id)" />
    <param name="trust_approx_solution" value="true" />
    <param name="publish_traversable" value="true" />
    <param name="verbose" value="true" />
    <param name="bb_dim_x" value="5.218" />
    <param name="bb_dim_y" value="2.8" />
    <param name="bb_dim_z" value="1.0" />
    <param name="reedsshepp" value="true"/>
    <param name="turning_radius_rs" value="11.0" />
    <param name="simplify_solution" value="true" />
    <param name="distance_between_poses" value="0.5" />
    <param name="search_radius_nn" value="0.225" />
    <param name="clearance" value="0.4" />
    <param name="ratio_traversables" value="0.8" />
    <param name="relax_neighbor_search" value="true" />
    <param name="state_validity_res" value="4.0" />
    <param name="wheelbase" value="2.99" />
    <param name="track" value="1.38" />
  </node>

</launch>
